### PR TITLE
Use whotracksme with ES module exports

### PR DESCRIPTION
**Pull request by smalluban**
Updates (for now, from the GitHub directly - DO NOT MERGE before updating to the npm version) `webextension-packages` with the ES module exports for the `serp-report`.

### CODE REVIEW

**Comment by smalluban** [comment]
Tested in Chrome Canary v99:
Zrzut ekranu 2022-01-26 o 16 23 11

Tested in Firefox v96:
Zrzut ekranu 2022-01-26 o 16 23 36

**Review by chrmod — COMMENTED** [review]
nicely done! 

lets wait for `@whotracksme/webextension-packages` release before merging

**Review by chrmod — APPROVED** [review]

**Pull request merged**
